src/NLog.Targets.Syslog/Policies/Throttling.cs

```diff
@@ -61,7 +61,10 @@ private void ApplyDeferment(int waitingLogEntries)
 
             var deferment = FixedTime(Delay, waitingLogEntries);
             InternalLogger.Warn("Applying defer throttling strategy ({0} ms)", deferment);
-            Thread.SpinWait(deferment);
+            if (deferment <= 0)
+                Thread.Sleep(1);
+            else
+                Thread.Sleep(deferment);
```

Comment:
I understand your point but the library user should use sensible params.
The rational in my opinion is: if you decide, why 1 and not 2 or 5?